src/coreComponents/mesh/generators/VTKMeshGenerator.hpp

```diff
@@ -0,0 +1,269 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2018-2020 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2020 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2018-2020 Total, S.A
+ * Copyright (c) 2019-     GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+/**
+ * @file VTKMeshGenerator.hpp
+ */
+
+#ifndef GEOSX_MESHUTILITIES_VTKMESHGENERATOR_HPP
+#define GEOSX_MESHUTILITIES_VTKMESHGENERATOR_HPP
+
+#include "dataRepository/Group.hpp"
+#include "codingUtilities/Utilities.hpp"
+#include "codingUtilities/StringUtilities.hpp"
+
+#include "MeshGeneratorBase.hpp"
+#include "mesh/CellBlockManager.hpp"
+
+#include <vtkSmartPointer.h>
+#include <vtkUnstructuredGrid.h>
+#ifdef GEOSX_USE_MPI
+#include <vtkMPIController.h>
+#include <vtkMPI.h>
+#else
+#include <vtkDummyController.h>
```

Comment:
If you follow my suggestion on making private member functions into non-member (free) functions in the cpp, you can remove most of these includes. I have to look more into VTK to see if we can store the mesh in `std::unique_ptr` instead of `vtkSmartPointer`, in which case all of these can be removed.